src/neo/SmartContract/BinarySerializer.cs

```diff
@@ -40,41 +40,35 @@ private static StackItem Deserialize(BinaryReader reader, uint maxItemSize, Refe
                 StackItemType type = (StackItemType)reader.ReadByte();
                 switch (type)
                 {
-                    case StackItemType.ByteArray:
-                        deserialized.Push(new ByteArray(reader.ReadVarBytes((int)maxItemSize)));
-                        break;
-                    case StackItemType.Boolean:
-                        deserialized.Push(new Boolean(reader.ReadBoolean()));
+                    case StackItemType.Any:
+                        deserialized.Push(StackItem.Null);
```

Comment:
Since `StackItemType.Any` is not a real type, how about renaming `StackItemType.Any` to `StackItemType.Void`?

Reply:
They are different.